frontend/src/scenes/session-recordings/player/PlayerController.tsx

```diff
@@ -49,6 +49,10 @@ export function PlayerController(): JSX.Element {
                     ))}
                 </Select.OptGroup>
             </Select>
+            <div className="rrweb-inactivity-toggle">
+                <span className="inactivity-label">Skip inactivity</span>
+                <Switch checked={skipInactivitySetting} onChange={setSkipInactivitySetting} size="small" />
+            </div>
```

Comment:
The "skip inactivity" label could also be a click trigger for the switch.

Reply:
Good call - making the change now.